docs/references/javascript/clerk/user-profile.mdx

```diff
@@ -243,10 +243,10 @@ All props below are optional.
 | Name | Type | Description |
 | --- | --- | --- |
 | `appearance` | <code>[Appearance](/docs/components/customization/overview) \| undefined</code> | Optional object to style your components. Will only affect [Clerk Components][components-ref] and not [Account Portal][ap-ref] pages. |
-| `routing` | `'hash' \| 'path' \| 'virtual'` | The routing strategy for your pages. <br />Supported values are: <ul><li>`hash` (default): Hash-based routing.</li> <li>`path`: Path-based routing.</li> <li>`virtual`: Virtual-based routing.</li></ul> |
+| `routing` | `'hash' \| 'path' \| 'virtual'` | The routing strategy for your pages.<br />Note: If you are using environment variables for Next.js or Remix to specify your routes, this will be set to `path`. |

```

Comment:
I don't think this note is applicable to to this component. AFAIK only sign in and sign up are impacted.